### PR TITLE
Refactor trach

### PR DESCRIPTION
**Pull request by VolodymyrTrach**

### CODE REVIEW

File: fn/client/src/components/product-list-paginator/Product-list-paginator.js [hunk 1]
```diff
@@ -7,24 +7,25 @@ function ProductListPaginator({ pagesCount, paginate, currentPage }) {
     for (let i = 1; i <= pagesCount; i += 1) {
         pageNumbers.push(i);
     }
+    const pageNumbersView = pageNumbers.map(number => {                    
+        let activeClass = ''
```
Comment (oleksandrhodzevych) [inline fn/client/src/components/product-list-paginator/Product-list-paginator.js:11]: ; missed

File: fn/client/src/components/checkout-form/checkout-form.js [hunk 2]
```diff
@@ -109,28 +80,56 @@ const CheckoutForm = ({
...
+            if (inCart.quantity > available) {
+                notAvaliable.push({
+                    available,
+                    name: inCart.title
+                })
+            }
+        });
+
+        if (notAvaliable.length) {
+            const snackbarText = notAvaliable.map((badItem) => {
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply fn/client/src/components/checkout-form/checkout-form.js]: renamed badItem to notAvailableItem

File: fn/client/src/components/checkout-form/checkout-form.js [hunk 3]
```diff
@@ -109,28 +80,56 @@ const CheckoutForm = ({
...
+            const { available } = product[0];
+            if (inCart.quantity > available) {
+                notAvaliable.push({
+                    available,
+                    name: inCart.title
+                })
+            }
+        });
+
+        if (notAvaliable.length) {
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply fn/client/src/components/checkout-form/checkout-form.js]: renamed

File: fn/client/src/components/checkout-form/checkout-form.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply fn/client/src/components/checkout-form/checkout-form.js]: renamed snackbarText as snackbarMessages

File: fn/client/src/components/checkout-form/checkout-form.js [hunk 4]
```diff
@@ -109,28 +80,56 @@ const CheckoutForm = ({
...
+            const snackbarText = notAvaliable.map((badItem) => {
+                return snackBarMsg(badItem);
+            })
+            snackbarHandler(snackbarText)
+            return
+        }
+        setsuccessOrder(true);
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply fn/client/src/components/checkout-form/checkout-form.js]: renamed setsuccessOrder as setSuccessOrder

File: fn/client/src/components/checkout-table/Checkout-table.js [hunk 5]
```diff
@@ -6,16 +6,34 @@ import { connect } from 'react-redux';
 
 const CheckoutTable = ({ cartProducts, currency, currencyIcon}) => {
 
+    // Count one row total price
     const tableRowPrice = (price , quantity) => {
         return (parseFloat(price * quantity * currency).toFixed(2))
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply fn/client/src/components/checkout-table/Checkout-table.js]: added all ;

File: fn/client/src/components/checkout-table/Checkout-table.js [hunk 6]
```diff
@@ -6,16 +6,34 @@ import { connect } from 'react-redux';
...
     }
 
+    // Count total price
     const totalCounter = () => {
         const row = cartProducts.map(product => product.price * product.quantity);
         const total = row.reduce((sum, next) => sum + next)
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply fn/client/src/components/checkout-table/Checkout-table.js]: added all ;

File: fn/client/src/components/checkout-table/Checkout-table.js [hunk 7]
```diff
@@ -6,16 +6,34 @@ import { connect } from 'react-redux';
...
         return (parseFloat(total * currency).toFixed(2))
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply fn/client/src/components/checkout-table/Checkout-table.js]: added all ;

File: fn/client/src/components/product-list-paginator/Product-list-paginator.js (hunk 1 above)
Comment (VolodymyrTrach) [reply fn/client/src/components/product-list-paginator/Product-list-paginator.js]: added all ;

**Pull request merged**